tests/test_download_puller.py

```diff
@@ -0,0 +1,112 @@
+import os
```

Comment:
I suggest that this file is split in two.

- test_plugin_helper.py
- test_plugins.py (which later can be split into separate files when needed)

Reply:
ok yes good deal -- I am with you